components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodProjectManager.kt

```diff
@@ -65,17 +79,15 @@ class GitpodProjectManager(
                 val projectRootManager = ProjectRootManager.getInstance(project)
                 if (projectRootManager.projectSdk == null) {
                     projectRootManager.projectSdk = sdk
-                    thisLogger().warn("gitpod: '${project.name}' project: SDK was auto preconfigured: $sdk")
```

Comment:
@felladrin I meant only to drop `SDK was already configured`, the fact when we really configure is useful

Reply:
They're back now!